### DISCUSSION TITLE
Projects not listing dependencies anymore

### DISCUSSION THREAD

**Discussion by ivirshup** [community/community#23938] [New to GitHub]
<p>Hi!</p>
<p>I’m just writing up some usage stats on my project, to find we no longer have any dependent packages! Given we had a few hundred a couple weeks ago, I really hope this isn’t the case.</p>
<p>Any idea what’s going on?</p>
<p>Here’s one of the projects and it’s current dependents page:</p>
<aside class="onebox githubfolder">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/theislab/scanpy/network/dependents" target="_blank" rel="noopener nofollow ugc">github.com</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/304;"><img src="https://user-images.githubusercontent.com/8238804/181080317-634ce124-e62d-44e2-8013-62a9bb66b077.png" class="thumbnail" width="690" height="304" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/6/c/6c350fd15345415518c409cbdf6b82d56276cd64_2_690x304.png, https://aws1.discourse-cdn.com/github/optimized/3X/6/c/6c350fd15345415518c409cbdf6b82d56276cd64_2_1035x456.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/6/c/6c350fd15345415518c409cbdf6b82d56276cd64_2_1380x608.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/6/c/6c350fd15345415518c409cbdf6b82d56276cd64_2_10x10.png"></div>

<h3><a href="https://github.com/theislab/scanpy/network/dependents" target="_blank" rel="noopener nofollow ugc">theislab/scanpy</a></h3>

<p><a href="https://github.com/theislab/scanpy/network/dependents" target="_blank" rel="noopener nofollow ugc">rk/dependents</a></p>

  <p><span class="label1">Single-Cell Analysis in Python. Scales to &amp;gt;1M cells.</span></p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>My current guess as to what’s happening:  (<em>update</em>: probably wrong, see below)</p>
<ul>
<li>We switched build tools (from <code>setuptools</code> to <code>flit</code>).</li>
<li>Now, github can’t parse our dependencies since they’re in <code>pyproject.toml</code>.</li>
<li>Somehow this means github also can’t figure out who depends on us?</li>
</ul>
<p>But this is just conjecture, which I’ll try to investigate more. Would really appreciate any insight on what to do here!</p>
<h2>Update:</h2>
<p>It looks like this isn’t working for other python projects like <a href="https://github.com/pandas-dev/pandas/network/dependents" rel="noopener nofollow ugc"><code>pandas</code></a>:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/0/c/0cb72c8245605a3e9408939b56d323f4cdce1cf8.png" data-download-href="/uploads/short-url/1Oub1BoXQsoqB5gm3m5eqK1yesU.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/8238804/181080321-3101b4ad-c3e7-495f-b87c-e9c8b58b58ae.png" alt="image" data-base62-sha1="1Oub1BoXQsoqB5gm3m5eqK1yesU" width="345" height="181" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/0/c/0cb72c8245605a3e9408939b56d323f4cdce1cf8_2_345x181.png, https://aws1.discourse-cdn.com/github/optimized/3X/0/c/0cb72c8245605a3e9408939b56d323f4cdce1cf8_2_517x271.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/0/c/0cb72c8245605a3e9408939b56d323f4cdce1cf8_2_690x362.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/0/c/0cb72c8245605a3e9408939b56d323f4cdce1cf8_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1242×654 48.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Seems like less of a configuration problem on our end and more like a github bug?</p>

**Comment by flying-sheep** [comment]
<p><a class="mention" href="/jurre">@jurre</a> <a class="mention" href="/feelepxyz">@feelepxyz</a> any idea?</p>

**Comment by ivirshup** [comment, marked as answer]
<p>Solved. Turns out something messed up on github’s end. I opened a ticket and it got fixed in a few hours. Something about “packages were no longer correctly associated with your repositories”.</p>
